package.json

```diff
@@ -16,10 +16,12 @@
     "@types/react-redux": "^7.1.7",
     "await-to-js": "^3.0.0",
     "classnames": "^2.3.1",
+    "decimal.js": "^10.3.1",
```

Comment:
There is also [a lighter version](https://github.com/MikeMcl/decimal.js-light/ ) which I think would be sufficient for this project.